.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "Listen for XDebug",
+			"type": "php",
+			"request": "launch",
+			"port": 9010,
+			"pathMappings": {
+				"/app/source/public": "${workspaceRoot}/app/public"
+			}
+		}
+	]
+}
```

Comment:
I'm a bit skeptical about including editor specific configuration to the code. If you have this locally you can gitignore it at `.git/info/exclude`.

Reply:
Ha, that's vestigial, I'll remove it. I'll add it to `.gitignore`

Reply:
File removed from version control!